### ISSUE TITLE
Evaluate 

### ISSUE THREAD

**Issue by MarijnJABoer** [naturalis/FormicID#19]
## Evaluation code
- [x] `model.evaluate()`
  - [link](https://github.com/keras-team/keras/issues/5140)
- [x] choose verbose
- [x] `evaluate_generator()` ?
- [x] evaluate on batch?
> Using model.evaluate will keep the model weights fixed and compute loss/accuracy for the whole data you give in. If you want to have the loss/accuracy on the training set, then you have to use model.evaluate and pass the training set to it. The history object does not have the true loss/accuracy on the training set.

> `.predict()` generates output predictions based on the input you pass it (for example, the predicted characters in the MNIST example)
`.evaluate()` computes the loss based on the input you pass it, along with any other metrics that you requested in the metrics param when you compiled your model (such as accuracy in the MNIST example)
## Metrics
- [x] Choose which metrics to print. Is only printing loss now
- [x] Accuracy
- [x] RMSE
